server/interactors/email.js

```diff
@@ -1,13 +1,21 @@
 var Mailchimp = require('mailchimp-api-v3'),
     config = require('../../config/default.js'),
-    mailchimp = new Mailchimp(config.mailchimp_token);
+    Responses = require('../responses/api/email');
 
-module.exports.ERRORS = {
+var Errors = {
     ALREADY_SUBSCRIBED: 'ALREADY_SUBSCRIBED',
+    MISSING_CONFIG: 'MISSING_CONFIG',
     UNKNOWN: 'UNKNOWN_ERROR'
 };
 
-module.exports.subscribe = function(email, callback) {
+var subscribe = function(email, callback) {
```

Comment:
If we're going this route, might as well just use `function subscribe(email, callback)`